content/manuals/enterprise/security/hardened-desktop/settings-management/configure-json-file.md

```diff
@@ -334,12 +354,13 @@ For more information, see [Networking](/manuals/desktop/features/networking.md#n
 
 ### AI settings
 
-| Parameter                                            | OS | Description                                                                                                                                                                                                                        | Version |
-|:-----------------------------------------------------|----|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
-| `enableInference`                                    |    | If `allowBetaFeatures` is true, setting `enableInference` to `true` enables [Docker Model Runner](/manuals/ai/model-runner/_index.md) by default. You can independently control this setting from the `allowBetaFeatures` setting. |         |
-| &nbsp; &nbsp; &nbsp; &nbsp; `enableInferenceTCP`     |    | Enable host-side TCP support. This setting requires Docker Model Runner setting to be enabled first.                                                                                                                               |         |
-| &nbsp; &nbsp; &nbsp; &nbsp; `enableInferenceTCPPort` |    | Specifies the exposed TCP port. This setting requires Docker Model Runner setting to be enabled first.                                                                                                                             |         |
-| &nbsp; &nbsp; &nbsp; &nbsp; `enableInferenceCORS`    |    | Specifies the allowed CORS origins. Empty string to deny all,`*` to accept all, or a list of comma-separated values. This setting requires Docker Model Runner setting to be enabled first.                                        |         |
+| Parameter                   | OS            | Description                                                                                                                                                                                                                         | Version |
+|:----------------------------|---------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `enableInference`           |               | Setting `enableInference` to `true` enables [Docker Model Runner](/manuals/ai/model-runner/_index.md).                                                                                                                              |         |
+| `enableInferenceTCP`        |               | Enable host-side TCP support. This setting requires Docker Model Runner setting to be enabled first.                                                                                                                                |         |
```

Comment:
```suggestion
| `enableInferenceTCP`        |               | Enable host-side TCP support. This setting requires the Docker Model Runner setting to be enabled first.                                                                                                                                |         |
```
Similar comments for below - or explicitly name the setting, e.g. "This setting requires the `enableInference` setting to be enabled first."

Reply:
@aevesdocker WDYT?
> This setting requires the enableInference setting to be enabled first.